src/include/security/helps.rb

```diff
@@ -352,9 +331,47 @@ def initialize_security_helps(include_target)
         "EXTRA_SERVICES"                            => _(
           "<P>Every running service is a potential target of a security attack. Therefore it is recommended to turn off all services which are not used by the system.</P>"
         )
-      } 
+      }
 
+    end
+
+    def boot_dialog_help
+      help = _(
+        "<p><b><big>Boot Security</big></b></p>\n<p>In this dialog, change various boot settings related to security.</p>"
+      )
+
+      # Boot dialog help 2/4, not shown for s390 architecture
+      if Arch.s390
+        help += _(
```

Comment:
I did so in the closed one, but @kobliha wasn't sure about some kind of problems, so i replaced but maybe he mind other thing in the comment and not about the operator.